app-admin/pwman3/pwman3-0.9.9.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A lightweight password manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+PYTHON_COMPAT=( python3_5 python3_6 )
+inherit distutils-r1
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+sqlite mysql postgresql mongodb test"
+
+RDEPEND="
+	>=dev-python/cryptography-2.2.2[${PYTHON_USEDEP}]
+	>=dev-python/colorama-0.3.0[${PYTHON_USEDEP}]"
+
+DEPEND="
+	sqlite? ( dev-lang/python[sqlite] )
+	mongodb? ( dev-python/pymongo[${PYTHON_USEDEP}] )
```

Comment:
How does it make sense for this to be required at build time only?

Also, sort lexically.